### PR TITLE
Librarian QoL

### PR DESCRIPTION
**Pull request by dontbetank**
## About the PR 
I'm not highly experienced developer, non-native English speaker and far from understanding balance for this game so tweaks and comments are welcome

Right now librarian role is boring and badly written. It's like a passenger but worse. If you are actually trying to make a library with compositions you can not even get enough books for yourself (I've seen no more than 3 on map) so you have to work with papers. Also moving big amounts of books is boring and this game doesn't have specified container for them. So I prototyped bag for moving big amount of your works. Added this as a starting gear of role and in Fun! vending machine because there is no special access for librarian role. Also placed 1 book in starting gear so librarian can start working even on Box station.
**Media**

icon
Bag itself

dude
Appearance of bag on character

order
Cargo order for books

crate
Ordered crate and example contents

- [X] I have added screenshots/videos to this PR showcasing its changes ingame
- [ ] This PR does not require an ingame showcase

**Changelog**

:cl: Prentor
- add: Added bag for carrying books and cargo order to get more of them

### CODE REVIEW

File: Resources/Prototypes/tags.yml [hunk 1]
```diff
@@ -593,3 +593,6 @@
 
 - type: Tag
   id: SubdermalImplant
+
+- type: Tag
+  id: Book
```
Comment (EmoGarbage404) [inline Resources/Prototypes/tags.yml:598]: Please place this alphabetically.

**Comment by Jackrost** [comment]
Maybe make book trolley like in IRL? You can copy functional from Janitor trolley

**Comment by dontbetank** [comment]
> Maybe make book trolley like in IRL? You can copy functionality from Janitor trolley

Janitor trolleys are currently holders for three items in one hand to pull but I get the point. Adding inventory to trolley looks better than bags but this requires some map changes which I don't want to perform.

**Comment by moonheart08** [comment]
just implement a book trolly anyway and people will put it into maps.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**